features-json/css-conic-gradients.json

```diff
@@ -334,7 +335,8 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Can be enabled via the \"Experimental Web Platform Features\" flag"
+    "1":"Can be enabled via the \"Experimental Web Platform Features\" flag",
+    "2":"Does not support multi-possition color stops"
```

Comment:
Is it necessary at all. Because this applies to all gradient types, not just conic ones. Thanks